Libraries/src/Amazon.Lambda.Core/Amazon.Lambda.Core.csproj

```diff
@@ -15,6 +15,12 @@
 	
   <ItemGroup>
 	<None Include="README.md" Pack="true" PackagePath="\"/>
-  </ItemGroup>	
+  </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
```

Comment:
I've removed this and re-worked the unit tests for `LambdaLoggerExtensions`. Now it will only test public methods. One downside is that we lost the test case for caching behavior of `MessageFormatter` instances since that class is internal.

I'm okay with not having that test case but if it's needed maybe the CI build can be changed to include am MSBuild  property like `<IsTest>true<IsTest>` and the `InternalsVisibleToAttribute` can become conditional?